Makefile

```diff
@@ -17,9 +17,7 @@ ci:
 	@echo "Build project"
 	@npm run build
 
-deploy: configure-docker deploy-image run-migrations
-	@kubectl -n gitlint-bot apply -f ./deploy/k8s-deployment.yaml
-	@kubectl -n gitlint-bot set image deployment/gitlint-bot-production gitlint-bot-production=${APP_IMAGE_SHA}
+deploy: decrypt-blackbox configure-docker deploy-image deploy-chart
```

Comment:
This indicates the dependent targets can be run in parallel (which should not be the case). Either [add a pipe](https://stackoverflow.com/questions/22630996/makefile-sequential-execution), or use `$(MAKE) [target]` for each targets.